client/build/asset-manifest.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
You should exclude all the build folder from git, as are files that can be generated locally

Reply:
Done, added docker execution instructions and update for the local